### PR TITLE
Creation of Journal event.

### PR DESCRIPTION
**Pull request by umair-nasir14**
Added journal event type in eventConfig page and allowed journal events to be created. Replaced continuous journal to journal for consistency across the code base.

### CODE REVIEW

**Review by avishkar58 — CHANGES_REQUESTED** [review]
Hey @umair-nasir14 , thanks for the PR and apologies for taking long to review it! 
This is a really great start and it's fantastic to simplify things and remove the distinction between "JOURNAL" and "CONTINUOUS_JOURNAL". I've left a few pointers to things you wouldn't have known about, but which need to be updated before we can submit this please. Thanks!

**Comment by umair-nasir14** [comment]
Hi @avishkar58! Thank you for your kind words and great review. I have catered to the changes you advised. I apologize for changing the older migrations. Now I know to check after changing through general search 😄

File: api/migrations/versions/2d7f5a0819e3_.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""empty message
```
Comment (avishkar58) [inline api/migrations/versions/2d7f5a0819e3_.py:1]: Please add a descriptive message

File: api/migrations/versions/2d7f5a0819e3_.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('answer_question_lookup', table_name='answer')
+    op.drop_index('answer_response_lookup', table_name='answer')
+    op.alter_column('event', 'event_type',
```
Comment (avishkar58) [inline api/migrations/versions/2d7f5a0819e3_.py:21]: This existing_type doesn't include the CONTINUOUS_JOURNAL which is in the current database and there is a bunch of generated lines in here that shouldnt' be like all these drop_index entries - so i suspect what's happened is that this is a result of generating the migration when your database was in an invalid state. This was most likely caused by the editing of old migrations. It's possible to do a series of migration downgrades to get your database back to an old state pre-edits, delete this migration, upgrade again, and then generate this migration again. But what I suggest doing instead is delete this migration. delete your db docker container and build a new one, then apply all the migrations with an upgrade, then generate this migration again - that way you can be sure it was generated by comparing to the proper latest database state.

**Review by avishkar58 — COMMENTED** [review]
Thanks for the changes! Unfortunately it looks like the new migration you've added was possibly generated from an invalid database state, because there are a bunch of entries in there that are not expected.

File: api/migrations/versions/2d7f5a0819e3_.py (hunk 2 above)
Comment (umair-nasir14) [reply api/migrations/versions/2d7f5a0819e3_.py]: I have followed your instructions and after migrating and upgrading, no errors persist. Please let me know if there is something to be addressed.

**Review by avishkar58 — APPROVED** [review]
Thanks for the changes! It looks great - will merge it in now :)

**Comment by umair-nasir14** [comment]
Thank you very much @avishkar58! :)

**Pull request merged**